### PR TITLE
icelake: initial configuration

### PR DESCRIPTION
**Pull request by jajanusz**
Added intial configuration for building icl firmware.
I can split it into more commits, but it will break fw build.

I guess you can ignore all src/icelake/include/arch/xtensa/config/* files, because they are literally copy-paste of cnl platform config - they should be part of toolchain anyway because they are going to generate a lot of boilerplate code in our repo for every platform if we will keep it here.

### CODE REVIEW

File: src/platform/icelake/base_module.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
```
Comment (lgirdwood) [inline src/platform/icelake/base_module.c]: Janusz Jankowski <janusz.jankowski@linux.intel.com> is the author :)

**Comment by lgirdwood** [comment]
@jajanusz can you update scripts folder to build icelake too.I'm assuming the xtensa ISA is the same as CNL ?

File: src/platform/icelake/base_module.c (hunk 1 above)
Comment (mUfoq) [reply src/platform/icelake/base_module.c]: I just changed 3 letters ;)
Should I add myself as author in every file that I touch - even if there is no meaningful input from my side?

Comment (lgirdwood) [reply src/platform/icelake/base_module.c]: I think it's better to if you create a new file or change > 5% lines of an existing file.

**Pull request merged**